server/views/referrals/new/courseParticipations/course.njk

```diff
@@ -24,9 +24,9 @@
     name: "otherCourseName",
     value: formValues.otherCourseName,
     type: "text",
-    classes: "govuk-!-width-one-third",
+    classes: "govuk-input--width-20",
```

Comment:
Looks like this is a bit wider on the prototype, using `govuk-!-width-two-thirds` - have you tried that here? Would probably make sense to be a bit wider

Reply:
conditionally-revealed setting and outcome inputs  > already done, I checked them.

Reply:
I'll double check as I recall the year already being 4 chars long.